### ISSUE TITLE
Questions about training from scratch

### ISSUE THREAD

**Issue by Levent9** [alibaba-damo-academy/FunCodec#31]
Hello, I followed the steps in run.sh to train with the LibriTTS-R dataset. Below is the training process loss. When I use the current checkpoint to synthesize speech, it is almost noise. Based on the loss, does the training of the model appear normal? Thank you! 

Loss:
![loss](https://github.com/alibaba-damo-academy/FunCodec/assets/110319697/94b19ea9-d01d-4ae1-9f17-aed7ba042824)
nll_loss:
![nll_loss](https://github.com/alibaba-damo-academy/FunCodec/assets/110319697/50477f2e-be51-489d-a555-5644224dbc92)
reg_l1_loss:
![reg_l1_loss](https://github.com/alibaba-damo-academy/FunCodec/assets/110319697/0053df6f-dc35-4f80-95a6-72b2437743da)
reg_l2_loss:
![reg_l2_loss](https://github.com/alibaba-damo-academy/FunCodec/assets/110319697/4d4f46e1-3db2-4cbb-9cbe-4b7606f5b5d8)

**Comment by Levent9** [comment]
When training LuaraGPT from scratch, the codebook in the model is not pre-trained？

**Comment by ZhihaoDU** [comment]
I think the training process is normal. On LibriTTS corpus, I got losses like these after about 106K iterations:
```
loss=6.557
nll_loss=4.271
reg_loss=2.286
reg_l1_loss=1.733
reg_l2_loss=2.840
out_acc_1=0.167
out_acc_2=0.100
```

If the synthesized speech is almost noise, there may be other bugs ?

**Comment by Levent9** [comment]
Thank you very much for your response. The text-to-speech synthesis is now functioning normally.
